src/app/globals.css

```
@@ -1,122 +1,259 @@
+
+.react-grid-layout .react-grid-item > .react-resizable-handle {
+    position: absolute;
+    width: 20px;
+    height: 20px;
+    bottom: 0;
+    right: 0;
+    background: hsl(var(--primary));
+    padding: 0 3px 3px 0;
+    background-image: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNiIgaGVpZ2h0PSI2IiB2aWV3Qm94PSIwIDAgNiA2IiBmaWxsPSJub25lIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPgo8ZG90cyBmaWxsPSJjdXJyZW50Q29sb3IiIGQ9Im0xNSA0LjEwOWMuNDA2LS4zMTYuNDA2LS45MDIgMC0xLjIxOEwxMS4yMTguNDM3QTEuMDAyIDEuMDAyIDAgMDAxMCAuMDQ2SDRhMSAxIDAgMDAtMS0xaC00YTEgMSAwIDAwLTEgMXY0aDJWMGgxdjRIMHYyaDF2NGgtMXY0aDJWNEgwVjJoMVYwaC0xdjRoMlY0YTEgMSAwIDAwMS0xaDQiLz4KPC9zdmc+');
+    background-position: bottom right;
+    padding: 0 3px 3px 0;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

padding is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)